package.json

```diff
@@ -123,6 +126,23 @@
     "ws": ">=3.3.1"
   },
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "objectExplorer",
+          "title": "Object Explorer",
+          "icon": "./images/server_page_inverse.svg"
+        }
```

Comment:
Does this not support light/dark images?

Reply:
[https://code.visualstudio.com/api/references/contribution-points#contributes.viewsContainers](https://code.visualstudio.com/api/references/contribution-points#contributes.viewsContainers) Looks like not - just double check this to be sure though if you wouldn't mind

---
In reply to: [303623862](https://github.com/microsoft/vscode-mssql/pull/1259#discussion_r303623862) [](ancestors = 303623862)